color_config:
  dilatation_kernel_size: 3
  canny_threshold: [80, 200]
  hough_threshold: 20
  hough_min_line_length: 3
  hough_max_line_gap: 1
  sobel_threshold: 40

  hsv_white1:  [0,0,150]
  hsv_white2:  [180,50,255]
  hsv_yellow1: [25,120,90]
  hsv_yellow2: [45,255,255]
  hsv_red1:    [0,140,100]
  hsv_red2:    [15,255,255]
  hsv_red3:    [165,140,100]
  hsv_red4:    [180,255,255]

